client/project-router.js

```diff
@@ -108,7 +108,7 @@ const ProjectRouter = () => {
           title={version.title || 'Untitled project'}
           subtitle="Project licence"
           basename={downloadBasename}
-          licenceStatus={isGranted ? project.status : 'inactive'}
+          licenceStatus={project.status === 'active' && !isGranted ? 'inactive' : project.status }
```

Comment:
I _think_ this is what you mean, but it's not obvious from the code as it is.

```suggestion
          licenceStatus={ (project.status === 'active' && !isGranted) ? 'inactive' : project.status }
```